frontend/packages/data-portal/app/components/MLChallenge/MainContent.tsx

```diff
@@ -300,7 +304,22 @@ export function MainContent() {
           </div>
         </div>
       </Section>
-      {/* TODO: Contact Banner */}
+
+      <Section className="font-semibold py-16" color="primary100">
```

Comment:
to be in line with the design:
```suggestion
      <Section className="font-semibold py-[50px]" color="primary100">
```